data/reportGeneration/studentToCoursesScript.php

```diff
@@ -0,0 +1,47 @@
+<?php
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    require_once('../../config/config.php');
+    require_once('../../business/login/passwordUtils.php');
+
+    $conn = new mysqli(DB_SERVER, DB_USER, DB_PASS, DB_NAME);
+
+    if ($conn->connect_error) {
+        die("Connection failed: " . $conn->connect_error);
+    }
+
+    $date = date('d-m-y-s');
+    $file = "report_courseList". $date .".txt";
```

Comment:
For the course report, I'd recommend including the course code as part of the filename

Reply:
Good idea, will do.